spec/std/int_spec.cr

```diff
@@ -437,7 +437,7 @@ describe "Int" do
   end
 
   it "raises when divides by zero" do
-    expect_raises(DivisionByZeroError) { 1 / 0 }
+    (1 / 0).should eq(Float32::INFINITY)
```

Comment:
Should this be `Float64::INFINITY`?

Reply:
Updated, and also change the description of the specs that used to raise.